zebra-consensus/src/block/check.rs

```diff
@@ -321,11 +321,45 @@ pub fn coinbase_expiry_height(
     }
 }
 
+/// Returns `Ok(())` if the expiry height for a non coinbase transaction is valid
+/// according to specifications [7.1] and [ZIP-203].
+///
+/// [7.1]: https://zips.z.cash/protocol/protocol.pdf#txnencodingandconsensus
+/// [ZIP-203]: https://zips.z.cash/zip-0203
+pub fn non_coinbase_expiry_height(
+    height: &Height,
```

Comment:
Optional: It's hard to tell which height we mean by `height`.

Can you rename the block height arguments in all 3 expiry height functions?

```rust
    block_height: &Height,
```